Cargo.toml

```diff
@@ -25,7 +25,7 @@ serde_cbor = "0.11.2"
 serde_json = "1.0.107"
 serde_qs = "0.12.0"
 serde_urlencoded = "0.7.1"
-ssi = "0.7.0"
+ssi = "0.8.1"
```

Comment:
Would be great if you could also refactor to just use the specific sub-crates that we need in this repo to contract the dependency tree, but feel free to do this once everything is implemented and working.